plugins/modules/azure_rm_sqldatabase.py

```diff
@@ -224,6 +224,7 @@
 
 import time
 from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase, format_resource_id
+import dateutil.parser
 
 try:
     from msrestazure.azure_exceptions import CloudError
```

Comment:
@nbr23 Please move the imported dateutil to the location below. Thank you very much!
```suggestion
    import dateutil.parser
    from msrestazure.azure_exceptions import CloudError
```